src/uncontrast/pages/UploadNewPaper/ExperimentsSection/Samples.jsx

```diff
@@ -0,0 +1,323 @@
+    );
+  };
+
+  console.log(
+    fieldValues[0].excluded === "yes"
+      ? parseInt(fieldValues[0]?.excludedNum) > 0
+      : null
+  );
+  const fieldsNum = fieldValues.filter((field) => field.id)?.length;
+  useEffect(() => {
+    setMinimumClassifications({
+      ...minimumClassifications,
+      samples: fieldsNum,
+    });
+  }, [fieldsNum]);
+  return (
+    <ExpandingBox
+      number={fieldsNum}
+      disabled={disabled}
+      headline={rawTextToShow(classificationName)}>
+      {fieldValues.map((fieldValue, index) => {
+        return (
+          <div key={`${classificationName}-${index}`}>
+            <form className="flex flex-col gap-2">
+              <div className="flex gap-2 items-center  border border-blue border-x-4 p-2 rounded-md">
+                <CircledIndex index={index} />
+                <div className="flex flex-col">
+                  <div className="w-full flex gap-2 items-start">
+                    <div className="w-full">
+                      <TooltipExplanation
+                        isHeadline
+                        tooltip={
+                          "Choose the type of population/s used in the experiment."
+                        }
+                        text={"Type"}
+                      />
+                      <CustomSelect
+                        disabled={fieldValue.id}
+                        value={fieldValue.type}
+                        onChange={(value) => {
+                          const newArray = [...fieldValues];
+                          newArray[index].type = value;
+                          setFieldValues(newArray);
+                          submitCondition(index) &&
+                            handleSubmit(fieldValues, index);
+                        }}
+                        options={fieldOptions}
+                      />
+                    </div>
+
+                    <div className="w-full">
+                      <TooltipExplanation
+                        isHeadline
+                        tooltip={
+                          "Enter the number of individuals from the specified population who participated in the experiment."
+                        }
+                        text={"Total"}
+                      />
+
+                      <div className="flex gap-2">
+                        <input
+                          min={0}
+                          disabled={fieldValues[index].id}
+                          type="number"
+                          value={fieldValue.total_size}
+                          onChange={(e) => {
+                            setFieldValues((prev) =>
+                              prev.map((item, i) =>
+                                i === index
+                                  ? { ...item, total_size: e.target.value }
+                                  : item
+                              )
+                            );
+                          }}
+                          onBlur={() =>
```

Comment:
Why so many event? You want it to auto save?
Maybe put the auto save logic in one function so there won't be differences between the events that fire the save action